### PR TITLE
docs: improving examples and docs

### PR DESCRIPTION
**Pull request by caarlos0**

### CODE REVIEW

File: examples/pty/main.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+		// Wish can allocate a PTY per user session.
+		ssh.AllocatePty(),
+
+		wish.WithMiddleware(
+			func(next ssh.Handler) ssh.Handler {
+				return func(sess ssh.Session) {
+					pty, _, _ := sess.Pty()
+					wish.Printf(sess, "Hello, world!\r\n")
+					wish.Printf(sess, "Term: "+pty.Term+"\r\n")
+					wish.Printf(sess, "PTY: "+pty.Slave.Name()+"\r\n")
```
Comment (aymanbagabas) [inline examples/pty/main.go:38]: This wouldn't work on Windows

File: examples/pty/main.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
+package main
```
Comment (aymanbagabas) [inline examples/pty/main.go:1]:
```suggestion
//go:build !windows

package main
```

**Review by aymanbagabas — APPROVED** [review]
Awesome! Few gotchas, looks good otherwise :)

File: examples/pty/main.go (hunk 1 above)
Comment (caarlos0) [reply examples/pty/main.go:40]: good catch!

**Pull request merged**